.gitignore

```diff
@@ -65,6 +65,7 @@ Thumbs.db
 .che
 dockerfiles/che/eclipse-che.tar.gz
 dockerfiles/che/eclipse-che/
+dockerfiles/theia/e2e/.Dockerfile
```

Comment:
It is not related to `Use bash instead of sh`. Can you elaborate on why you did this change?

Reply:
yeah, this file can be ignored as it is generated during the build and if build fails this file remain in the repo.

Reply:
It's kind of bug in `build.include`. And this issue appears during a build of all images - Che-Server, Postgres, Che-Dir, etc. So, it should be fixed for all. Possible solution - name a generated file like `.Dockerfile.genarated` and add it to `.gitignore` with a regular expression.
Please revert your change in this PR and create a separate issue for this bug - welcome to fix it in proposed way ;)